packages/server/package.json

```diff
@@ -31,16 +31,19 @@
     "@nestjs/passport": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
     "@prisma/client": "^4.6.1",
+    "axios": "^1.2.0-alpha.1",
     "bcrypt": "^5.1.0",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "crypto-js": "^4.1.1",
     "date-fns": "^2.29.3",
     "dotenv": "^16.0.3",
     "joi": "^17.7.0",
+    "nodemailer": "^6.8.0",
     "passport": "^0.6.0",
     "passport-jwt": "^4.0.0",
     "passport-local": "^1.0.0",
+    "prisma": "^4.5.0",
```

Comment:
Do we need to have this in dependencies? @gregfrasco @cbolles

Reply:
removed in latest commit